packages/react-reconciler/src/__tests__/ReactHooksWithNoopRenderer-test.internal.js

```diff
@@ -3173,6 +3123,59 @@ function loadModules({
         expect(ReactNoop).toMatchRenderedOutput('5');
       });
 
+      it('should not store dispatched items in the queue (TODO: better description)', () => {
```

Comment:
I'm not sure how to best describe this test as it sort of tests the mechanism that is no longer here (eager bailouts for use reducer).